doc/source/whatsnew/v0.7.3.rst

```diff
@@ -3,13 +3,6 @@
 v.0.7.3 (April 12, 2012)
 ------------------------
 
-{{ header }}
```

Comment:
we should just leave the ``{{ header }}`` in each file for consistency.

Reply:
yes, didn't see that, please don't remove any header